### ISSUE TITLE
Easily check if response object is empty?

### ISSUE THREAD

**Issue by sethxd** [appetizermonster/hain#80]
I'm working on a plugin, and if the search doesn't return any results I wanted to add a search result that indicates that and tells them to search again, instead of not providing any feedback.

I feel like should be easy to check if there aren't currently any results, and then create this one, but not sure how to do it. Thanks for any help!

**Comment by Semigradsky** [comment]
Maybe something like this?
```js
  function search(query, res) {
    if (...) {
      res.add({
        id: 0,
        title: 'Please, search again',
        desc: '...'
      });
      return;
    }

    res.add({
      id: 42,
      title: '...',
      desc: '...',
      payload: '...'
    });
  }

  function execute(id, payload) {
    if (!payload) {
      return;  
    }

    ...
  }

```

**Comment by sethxd** [comment]
Thanks for the suggestion, I will give this a try!

**Comment by appetizermonster** [comment]
Fixed at 39f606009577536c410c0aa0ca31a2f2bd6c9030
